tests/data.py

```diff
@@ -25,7 +25,11 @@ def users_only() -> list[User]:
     ]
 
 
-def users() -> list[User]:
+def users(today: datetime.date = datetime.date(2002, 3, 12)) -> list[User]:
+
+    def since(days: int) -> datetime.date:
+        return today - datetime.timedelta(days)
```

Comment:
I think the name `since` is confusing, as `since` typically refers to a period of time. What do you think of using `days_ago` instead?

Reply:
That's better, changed.